python/cudf/cudf/pandas/module_accelerator.py

```diff
@@ -595,9 +595,10 @@ def install(
                 )
             mode = deduce_cudf_pandas_mode(slow_lib, fast_lib)
             if mode.use_fast_lib:
-                importlib.import_module(
+                pandas_wrappers = importlib.import_module(
                     f".._wrappers.{mode.slow_lib}", __name__
                 )
+                pandas_wrappers.cudf.set_option("mode.pandas_compatible", True)
```

Comment:
I think this is the right place to turn on pandas compatible mode and not inside of `._wrappers.pandas` module as it's a public module and we don't want to run into the situation we ran into today with `cuml`.

Reply:
I don't know if I would go so far as to call `_wrappers.pandas` public (hopefully nobody is importing it directly) but I definitely agree that this is the right place to put this setting.

Reply:
Sorry, yes it's not actually public.

Reply:
I do feel like this leaks cudf implementation details here, where `module_accelerator` is supposed to be agnostic of the fast and slow library. Where else would a user need to import from `._wrappers.pandas`?

Reply:
@mroeschke Do you think a reasonable thing to do is define a method in `pandas.py`:

````python
def setup_module():
    cudf.set_option("mode.pandas_compatible", True)
```

and then call `pandas_wrappers.setup_module` here?

Reply:
Sounds good, I'll let this PR merge and I have a fix for another cudf.pandas issue today. I'll incorporate this change there.